python/shark_turbine/dynamo/importer.py

```diff
@@ -373,6 +373,13 @@ def _import_torch_op_overload(
         if schema.overload_name != "":
             mlir_op_name += f".{schema.overload_name}"
 
+        # Intervening to use Scalar ops due to incorrect ops from AOT-autograd with scalar arguments.
+        if mlir_op_name in TENSOR_SCALAR_OP_CONVERTER:
+            if isinstance(node.args[1], float) or isinstance(node.args[1], int):
+                mlir_op_name = TENSOR_SCALAR_OP_CONVERTER.get(
```

Comment:
Nit: Sightly more efficient to use [] instead of get if you know it won't raise an exception.